allennlp/predictors/predictor.py

```diff
@@ -58,6 +61,92 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         instance = self._json_to_instance(inputs)
         return self.predict_instance(instance)
 
+    def saliency_interpret_from_json(self, inputs: JsonDict) -> JsonDict:
```

Comment:
Shouldn't this live on an `Interpreter`, not on `Predictor`?  Similar to how `Attacker` owns `attack_from_json`.